js/ui/paygUnlockDialog.js

```diff
@@ -0,0 +1,395 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
```

Comment:
Should this file have a copyright header?

Reply:
Yes, it should. I slipped here due to starting off unlockDialog.js, which does not have one, but I will add it